classes/Customer.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * Shopgate GmbH
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License (AFL 3.0)
+ * that is bundled with this package in the file AFL_license.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to interfaces@shopgate.com so we can send you a copy immediately.
+ *
+ * @author    Shopgate GmbH, Schloßstraße 10, 35510 Butzbach <interfaces@shopgate.com>
+ * @copyright Shopgate GmbH
+ * @license   http://opensource.org/licenses/AFL-3.0 Academic Free License ("AFL"), in the version 3.0
+ */
+
+if (version_compare(_PS_VERSION_, '1.5.0', '<')) {
+    include_once(_PS_MODULE_DIR_.'shopgate'.DIRECTORY_SEPARATOR.'classes'.DIRECTORY_SEPARATOR.'ModObjectModelCustomer.php');
+} else {
+    include_once(_PS_MODULE_DIR_.'shopgate'.DIRECTORY_SEPARATOR.'classes'.DIRECTORY_SEPARATOR.'ModObjectModelCustomerDummy.php');
```

Comment:
This won't give me the same result. I want to extend the ObjectModel in case its a Prestashop version <= 1.5.0.0 (see file ModObjectModelCustomer.php)